package.json

```diff
@@ -282,7 +301,7 @@
             ],
             "view/item/context": [
                 {
-                    "command": "azureResourceGroups.selectSubscriptions",
+                    "command": "azureResourceGroups..accounts.selectSubscriptions",
```

Comment:
extra `.` here?

Reply:
It's how Phillip originally wrote it, but we can always change it up.

Reply:
Oh wait, you're right, I could barely see that with the highlighting 👍